ansible/vars/topo_t0-52.yml

```diff
@@ -94,7 +94,7 @@ configuration_properties:
     dut_asn: 65100
```

Comment:
Is this topology only used by M0?

Reply:
I only see M0 using it in our repo.

Reply:
In future, we need to make this topology specific to mgmtToR so that we don't use this for T0